hdKeysGenerator/presenter.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+)
+
+func askMnemonic() (data.Mnemonic, error) {
+	fmt.Println("Enter an Elrond-compatible mnemonic:")
+	line, err := readLine()
+	if err != nil {
+		return "", err
+	}
+
+	mnemonic := data.Mnemonic(line)
```

Comment:
can we have one word mnemonic here and still work? I guess we need some validation here.